packages/web3-providers/src/coingecko/index.ts

```diff
@@ -2,9 +2,33 @@ import { CurrencyType, Price } from '@masknet/web3-shared-base'
 import type { PriceAPI } from '..'
 
 const COINGECKO_URL_BASE = 'https://api.coingecko.com/api/v3'
+const COINGECKO_CHAIN_ID_MAPPING: { [key: string]: string } = {
+    1: 'ethereum',
```

Comment:
```suggestion
    [ChainId.Mainnet]: 'ethereum',
```

Reply:
`ChainId` does not contain some ids, e.g. avalanche-2